### PR TITLE
Feat: Incident manager part 2

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
- updated incident manager UI
- Added cypress test (scenario 1 and scenario 2) from this [issue](https://github.com/open-metadata/OpenMetadata/issues/13746#issuecomment-1880879170), scenario 3 and scenario 4 will be cover in separate PR

image
image
image

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/IncidentManager/TestCaseIssuesTab/TestCaseIssueTab.component.tsx
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/IncidentManager/TestCaseIssuesTab/TestCaseIssueTab.component.tsx:1]: It's IncidentTab?

File: openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/TestSummaryCustomTooltip.component.tsx [hunk 1]
```diff
@@ -28,21 +36,57 @@ const TestSummaryCustomTooltip = (
...
-            key={`item-${key}`}>
-            <span className="flex items-center text-grey-muted">
-              {startCase(key)}
-            </span>
-            <span className="font-medium" data-testid={key}>
-              {value}
-            </span>
-          </li>
-        ))}
+      <ul data-testid="test-summary-tooltip-container">
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/TestSummaryCustomTooltip.component.tsx:43]: Can we use AntD List component here?

Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/TestSummaryCustomTooltip.component.tsx:94]: I will address this comment separately as I need to make similar changes in one more component. Thanks

**Comment by ShaileshParmar11** [comment]
@chirag-madlani I have pushed the review comments changes.

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**